### ISSUE TITLE
provide pkg-config files

### ISSUE THREAD

**Issue by nschloe** [TriBITSPub/TriBITS#213]
They seemed like an autotools relic, but apparently more modern configure systems (like [meson](http://mesonbuild.com/)) use them as well. Shouldn't be too hard to write a template for it.

**Comment by bartlettroscoe** [comment]
@nschloe, I think we are getting close to starting to work on #63 where we will be going over the `<Package>Config.cmake` files and that would be a good time to consider pkg-config files.  Is this something you could help with?  Otherwise, I don't know of any customers that would use this.

**Comment by nschloe** [comment]
> Is this something you could help with?

Certainly. Just wanted to make sure that this is something remotely mergeable.

**Comment by bartlettroscoe** [comment]
> >  Is this something you could help with?
>
> Certainly. Just wanted to make sure that this is something remotely mergeable.

Thanks.  We will also need good automated tests and documentation as well as per:

* https://github.com/TriBITSPub/TriBITS/blob/master/CONTRIBUTING.md

That is often the hardest part.
